devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -205,17 +201,18 @@ def configure() {
 	// enrolls with default periodic reporting until newer 5 min interval is confirmed
 	sendEvent(name: "checkInterval", value: 2 * 10 * 60 + 1 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
 
-	// OnOff minReportTime 0 seconds, maxReportTime 5 min. Reporting interval if no activity
+	// OnOff, level, colorTemperature minReportTime 0 seconds, maxReportTime 5 min. Reporting interval if no activity
+	zigbee.onOffConfig(0, 300) +
+	zigbee.levelConfig(0, 300) +
+	zigbee.colorTemperatureConfig(0, 300) +
```

Comment:
If I remember correctly we're not using reporting for color temperature and color on purpose. @varzac do you recall why?

Reply:
I don't remember any context.

Reply:
@tpmanley I added setting the color temperature to 6500K in configuration().
To double-check -> will those devices who support color temp up to < 6500K ignore that command or they should consume it and set max possible value ?

Reply:
The spec doesn't seem to say from my quick check. Personally I think this is fine for now and we can improve it later if we find it doesn't work when joining a device that doesn't support up to 6500.